src/components/ConnectWallet/ConnectWalletDialog.tsx

```diff
@@ -15,8 +15,10 @@ import { WalletConnectQR } from 'utils/WalletConnect'
 const NO_WALLET_HELP_CENTER_URL = 'https://help.uniswap.org/en/articles/5391585-how-to-get-a-wallet'
 
 const Body = styled(Column)`
+  display: grid;
+  gap: 12px;
```

Comment:
This gap looks much larger than the Row's horizontal gap. Can they be made equal?